lib/routes/account.js

```diff
@@ -56,7 +57,9 @@ module.exports = function (
   })
   const features = require('../features')(config)
 
-  var securityHistoryEnabled = config.securityHistory && config.securityHistory.enabled
+  const securityHistoryEnabled = config.securityHistory && config.securityHistory.enabled
```

Comment:
I have to add a `use strict` to the top of this file to stop receiving 

```
auth-server key server-2 fxa-auth-server.CRITICAL: undefined SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```
in Node 4.5